requirements.txt

```diff
@@ -29,6 +29,7 @@ unstructured
 json_repair
 json5
 loguru
+langchain-ibm==0.1.12
```

Comment:
Hey @barvhaim can you remove this line? The idea is to not have any optional dependencies. If and when someone will use it then a manual install will be necessary. Thanks!